x-pack/plugins/stack_connectors/server/connector_types/bedrock/render.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderParameterTemplates } from './render';
+import Mustache from 'mustache';
+
+const params = {
+  subAction: 'run',
+  subActionParams: {
+    body: '{"domain":"{{domain}}"}',
+  },
+};
+
+const variables = { domain: 'm0zepcuuu2' };
+
+describe('Bedrock - renderParameterTemplates', () => {
+  it('should not render body on test action', () => {
```

Comment:
tbh I'm not sure I understand the purpose of renderParameterTemplates, I was following other connector types... in case there are mustache variables used in the test/run json form?

Reply:
`renderParameterTemplates` is an optional function that you can register with the connector type if you need special escaping for some of the parameter values, typically while running or testing the action. This function as is is saying to only use this function for the run or test subaction and to not worry about mustache templating for any of the other subactions. Maybe you want to change it so it only renders the mustache templates during the run subaction? Then this unit test would make sense.

Reply:
the test and run action are the same